streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -983,6 +1120,27 @@ private void verifyRemovedTasks(final Task... tasks) throws Exception {
         }
     }
 
+    private void verifyPausedTasks(final Task... tasks) throws Exception {
```

Comment:
We also need to verify the paused tasks in the existing test methods. For instance, `shouldImmediatelyAddStatelessTasksToRestoredTasks()` should include the call `verifyPausedTasks()` to verify that a stateless task is not added to the paused tasks.

Reply:
Ack.